ko-KR.json

```diff
@@ -436,14 +436,14 @@
 	"SETTINGS_SHORTCUT_SPACE": "Space",
 	"SETTINGS_SHORTCUT_SHIFT": "Shift",
 	"SETTINGS_SHORTCUT_ESC": "Esc",
-	"SETTINGS_TRAKT_AUTHENTICATE": "Authenticate",
-	"SETTINGS_FACEBOOK_IMPORT": "Import",
-	"SETTINGS_CALENDAR_SUBSCRIBE": "Subscribe",
-	"SETTINGS_INTERFACE_LANGUAGE": "Interface Language",
+	"SETTINGS_TRAKT_AUTHENTICATE": "인증하기",
+	"SETTINGS_FACEBOOK_스트림": "가져오기",
```

Comment:
there is an issue here, "SETTINGS_FACEBOOK_IMPORT" should not be changed because it will break the translation in the apps

Reply:
I was making overall edits and it seems I got included in the edits as well. fixed it! thank you